guides/v2.3/graphql/reference/quote.md

```diff
@@ -27,12 +160,413 @@ mutation {
 
 **Response**
 
-The response is the quote ID, which is sometimes called the cart ID.
+The response is the quote ID, which is sometimes called the cart ID. The remaining examples in this topic will use this cart ID.
 
 ```json
 {
   "data": {
-    "createEmptyCart": "6XZA7q1ooLEI0jLz8DfFrfruEqgxGzlt"
+    "createEmptyCart": "4JQaNVJokOpFxrykGVvYrjhiNv9qt31C"}
   }
 }
 ```
+
+### Adding and removing coupons from a cart
+{:.no_toc}
+
+You can use mutations to add or remove coupons from a specified cart.
+
+### Coupon attributes
+The coupon object can contain the following attributes:
+
```

Comment:
I'm noticing there are at least 4 coupon objects, Apply...Input, Apply...Output, Remove...Input, and Remove...Output. Finesse this statement so that it doesn't sound like there's a monolithic coupon object.

Reply:
finessed to call out the add/remove coupon objects